src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -148,6 +161,30 @@ public void mouseReleased(final MouseEvent e) {
           notifyUnitSelected(tuple.getSecond(), tuple.getFirst(), md);
         }
       }
+
+      private boolean is4Pressed = false;
+      private boolean is5Pressed = false;
+      private int lastActive = -1;
+
+      @Override
+      public void mousePressed(final MouseEvent e) {
+        is4Pressed = e.getButton() == 4 ? true : is4Pressed;
+        is5Pressed = e.getButton() == 5 ? true : is5Pressed;
+        if (lastActive == -1) {
+          new Thread(() -> {
+            while (lastActive != -1) {
+              final int diffPixel = tripleAFrame.computeScrollSpeed(e);
+              if (lastActive == 5) {
+                setTopLeft(getXOffset() + diffPixel, getYOffset());
+              } else if (lastActive == 4) {
+                setTopLeft(getXOffset() - diffPixel, getYOffset());
+              }
+              ThreadUtil.sleep(50);
```

Comment:
We have a number of user settings for scroll speed. Should this be included with those? Why 50 millis? (The rationale for that likely should be captured in the code as a comment)